sycl-jit/jit-compiler/lib/translation/KernelTranslation.cpp

```diff
@@ -260,9 +260,8 @@ KernelTranslator::translateToPTX(SYCLKernelInfo &KernelInfo, llvm::Module &Mod,
 
   // Give priority to user specified values (through environment variables:
   // SYCL_JIT_AMDGCN_PTX_TARGET_CPU and SYCL_JIT_AMDGCN_PTX_TARGET_FEATURES).
-  llvm::StringRef CPU = ConfigHelper::get<option::JITTargetCPU>().begin();
-  llvm::StringRef Features =
-      ConfigHelper::get<option::JITTargetFeatures>().begin();
+  std::string CPU = ConfigHelper::get<option::JITTargetCPU>().begin();
+  std::string Features = ConfigHelper::get<option::JITTargetFeatures>().begin();
```

Comment:
```suggestion
  auto CPUVal = ConfigHelper::get<option::JITTargetCPU>();
  auto FeaturesVal = ConfigHelper::get<option::JITTargetFeatures>();
  llvm::StringRef CPU = CPUVal.begin();
  llvm::StringRef Features = FeaturesVal.begin();
```

Can we do something like this instead? (and below)

Reply:
Yeap, done.